Sources/TuistSupport/Utils/Environment.swift

```diff
@@ -82,6 +100,12 @@ public class Environment: Environmenting {
 
     /// Sets up the local environment.
     public func bootstrap() throws {
+        // load `.env` variables into `ProcessInfo.processInfo`
+        do {
+            try loadEnvironmentVariablesFromFile()
+        } catch {
+            logger.error("unable to load environment values from .env file with reason: \(String(describing: error))")
+        }
```

Comment:
If the logic errors, I'd not swallow the error, otherwise, developers will get unexpected results and spend time debugging the issue. We tend to lean on the strictness side when it comes to errors, so I'd suggest to do the same thing here. The error will eventually reach an error handler, who will take care of formatting and presenting it to the user.

Reply:
> The error will eventually reach an error handler, who will take care of formatting and presenting it to the user.

I don't think that's the case here because `Environment.bootstrap()` is called in `TuistCommand` whereas our `FatalError` handling wraps the command execution itself.

Reply:
In that case I'd adjust the error handler and move it to a different level so that we preserve the consistency.